<launch>

    <node pkg="cv_camera" type="cv_camera_node" name="cv_camera_node">
        <param name="rate" value="30"/>
        <param name="device_id" value="0"/>
        <param name="image_width" value="1280"/>
        <param name="image_height" value="720"/>

        <remap from="cv_camera_node/image_raw" to="image_raw"/>
    </node>

    <include file="$(find face_cropping)/launch/face_cropping.launch">
        <arg name="output" value="screen"/>

        <arg name="input_camera_topic" value="image_raw"/>
        <arg name="output_camera_topic" value="image_raw/cropped"/>
        <arg name="face_detection_model" value="lbpcascade"/>
    </include>

</launch>
